adops/step-by-step.md

```diff
@@ -1,9 +1,241 @@
+5. After you’ve created the copies, click into each one and change the **Name**. If you’ve followed our suggestions, you can name each one the same but append a subsequent number. For example, `Prebid – banner – 1x1 – 1`, `Prebid – banner – 1x1 – 2`, etc.
+
+![List of duplicated creatives](/assets/images/ad-ops/gam-sbs/duplicate-creatives.png)
+
+
+## Attach Creatives to Line Item
+
+Now we need to attach the creatives to your line item. Navigate to **Delivery** > **Line items** and select the line item you created earlier.
+
+Under the **Creatives** tab, you’ll see a yellow box showing each size you entered for your line item that doesn’t yet have a creative attached. For each size in the list, do the following:
+
+1. Click **Existing creative**. A message at the top of the screen will tell you that creatives have been filtered based on size. Click **Undo**.
+
+![Creative filter based on size](/assets/images/ad-ops/gam-sbs/creative-filter.png)
+
+{:start="2"}
+2. Click **Creatives**.
+3. Select one of the creatives you just created and click **Save**. (If you have more creatives than you do sizes, you can select multiple creatives before clicking Save.)
+
+![Select creatives to attach to line item](/assets/images/ad-ops/gam-sbs/select-creative.png)
+
+Repeat the preceding steps until all the sizes in your line item have creatives associated with them. When you’re done, the **Creatives** tab under your line item will show a list of all the associated creatives.
+
+
+## Duplicate Line Item
+
+You’ve now created a line item for one price bucket for a single bidder. Next you need to create line items for the rest of the price buckets for that bidder. The simplest way to do that (outside of automation) is to duplicate the line item you just created.
+
+1. From **Display** > **Line Items**, click the check box next to your line item.
+2. Select **Copy to** from the action bar that appears.
+3. In the **Copy line item** window, select **Copy and share creatives**, and copy to **Same order**.
+4. Input the number of copies you need to make to cover all price buckets for the bidder.
+5. Click **OK**.
+
+You now need to click into each line item to change the following values to reflect the new price bucket:
+
+- Rate
+- Name
+- hb_pb_BIDDERCODE key value
+- Long-form video only: hb_pb_cat_dur key value
+
+## Additional Bidders
+
+If you’re using a Send Top Price cofiguration, at this point you’re done. Congratulations!
+
+If you’re using a Send All Bids configuration, you need to repeat all the above steps for each of your bidders. You can copy one of the existing creatives and change the BIDDERCODE, and you can copy the existing line items and change the names, so you don’t have to completely start from scratch.
+
+## Video Tutorial
+
+<div id="youtube">
+<h2>(Sorry, YouTube videos aren't available with your cookie privacy settings.)</h2>
+<p><a class="optanon-show-settings">Cookie Settings</a></p><br />
+</div>
```

Comment:
This doesn't work in the preview.

Reply:
Let's just remove the video tutorial. It's way out of date. Philip is working on a new set of videos for the site.